api.ts

```diff
@@ -2264,6 +2337,7 @@ export function makeSyncTable<
       supportsUpdates: Boolean(executeUpdate),
```

Comment:
You probably need an equivalent of this, otherwise we don't know when looking at a pack's metadata whether there is an executeGetPermissions function to call (javascript gets stripped out of the metadata).

Reply:
Added this as well